tests/console/yast2_proxy.pm

```diff
@@ -189,19 +216,23 @@ sub run {
     # move to Start-Up and start proxy server now
     #	for (1..35) {send_key 'tab'; save_screenshot;}
     send_key_until_needlematch 'yast2_proxy_miscellanous_selected', 'shift-tab';
-    for (1 .. 7) { send_key 'up'; }
+    send_key_until_needlematch 'yast2_proxy_start-up',              'up';
+    wait_still_screen 1;
     send_key 'ret';
 
     assert_screen 'yast2_proxy_squid';
+    wait_still_screen 1;
     # now save settings and start squid server
     send_key 'alt-s';
     #	check again before to close configuration
     assert_screen 'yast2_proxy_before_close';
     # finish configuration with OK
-    send_key 'alt-o';
+    wait_screen_change { send_key 'alt-o' };
 
     # yast might take a while on sle12 due to suseconfig
-    wait_serial("yast2-squid-status-0", 60) || die "'yast2 squid' didn't finish";
+    wait_serial("yast2-squid-status-0", 180) || die "'yast2 squid' didn't finish";
+    type_string "clear\n";
+    check_screen 'root-console';
```

Comment:
The purpose of it is mainly to wait for a root-console, but as we may get distorted screen (what happens in the runs on production), we still should try to execute commands below. So, I'll remove this completely then, as it's a hack to make test results more stable.